pkg/jwt/jwtToken.go

```diff
@@ -0,0 +1,46 @@
+package jwt
+
+import (
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+type JwtIssuer struct {
+	issuer string
+	secret string
+	ttl    time.Time
+}
+
+func NewJwtIssuer(issuerId string,secretKey string,ttl time.Time) (*JwtIssuer) {
+	return &JwtIssuer{
+		issuer: issuerId,
+		secret:secretKey,
+		ttl: ttl,
+	}
+}
+func (j *JwtIssuer) GeneratedJwtToken(extras map[string]string) (string, error) {
```

Comment:
if you don't plan implement adding values from extras map I would remove it

Reply:
done remove if needed we will implement in future